fog/ingest/server/src/error.rs

```diff
@@ -127,6 +131,41 @@ impl From<GrpcError> for IngestServiceError {
     }
 }
 
+impl From<ReportParseError> for IngestServiceError {
+    fn from(src: ReportParseError) -> Self {
+        Self::ReportParse(src)
+    }
+}
+
+/// An error which occurs when parsing an AVR
+#[derive(Debug, Display)]
+pub enum ReportParseError {
+    /// Verification error: {0}
```

Comment:
I would drop the `error` suffix, it should be obvious when it is displayed that it is an error. Same for the other variant below.